apmapf.py

```diff
@@ -186,6 +261,11 @@ def run(self):
                                               i, INT)]
                       for i in range(H)]
 
+            # formula says that forall components of all coordinates in the
+            # attack, the plan is a valid plan, but the attack is not a valid
+            # attack.  So, a violation (which z3 would detect) would mean an
+            # attack.
```

Comment:
This comment in particular might be misleading.  Or, maybe the exact negation of the correct comment.  Clarification here would be great.  Thanks!

Reply:
There is an implicit "exists" quantifier out front on the `plan` variables. We are asking the solver to determine if there exists a model for the `plan` variables s.t. `plan` is a valid MAPF solution and s.t. forall choices (in (Z^2)^H) for the `attack` variables, the attack is not valid.

Reply:
Ok, that makes sense.  I just adopted your wording with minor modification (will push in a few minutes).  Thanks!